### ISSUE TITLE
Montenegro language change - not urgent

### ISSUE THREAD

**Issue by GrahamCSmith** [BiologicalRecordsCentre/MammalNet-Europe#96]
@kazlauskis Please accept the Montenegrin language request, copy all the text from Serbian (Montenegro) to the new language, close the old one, and ensure the web site and app picks up Montenegrin on future updates. 
This should be done before we make any more requests for translations from Montenegrin, so no rush at the moment, but this is obviously a sensitive issue locally.

**Comment by GrahamCSmith** [comment]
Remains a low priority with no current plans to change the overall text.
